tutor/interactive.py

```diff
@@ -148,6 +150,16 @@ def ask_questions(config: Config, run_for_prod: Optional[bool] = None) -> None:
             config,
             defaults,
         )
+    run_clean = click.confirm(
```

Comment:
This will make it so that the user is always prompted to start fresh, whether or not `-c` was provided. I do not think we want to ask the user this every they launch interactive config. I think it would encourage people to delete their environment too often out of caution, which isn't the indented regular Tutor workflow.

Can you change it so that they are only prompted if `-c` is provided? To make the implementation easier, you can simply abort if they say no. The prompt could be something like "The -c flag you provide will delete your existing Tutor environment. Are you sure you want to do this?"

Reply:
Totally agree. TBH, I really didn't know what exactly we wanted in interactive mode.